src/bubblesort.py

```diff
@@ -0,0 +1,11 @@
+"""Bubblesort sorting method."""
+
+
+def bubblesort(list):
```

Comment:
Be careful with variable names. `list` is a keyword built into python and shouldn't be overridden.